docs/src/pages/[platform]/theming/default-theme/colors/web.mdx

```diff
@@ -0,0 +1,94 @@
+import { Tabs, TabItem, View } from '@aws-amplify/ui-react';
+import { TokenList } from '@/components/Theming/TokenList';
+import { Example, ExampleCode } from '@/components/Example';
+import { ColorsExample } from './examples';
+
+<Tabs>
+  <TabItem title="Example usage">
+    <View marginTop="var(--amplify-space-large)">
+      <Example>
+        <ColorsExample  />
+        <ExampleCode>
+```tsx file=./examples/ColorsExample.tsx
+
+```
+        </ExampleCode>
+      </Example>
+    </View>
+  </TabItem>
+  <TabItem title="CSS">
+    <View marginTop="var(--amplify-space-large)">
+    <Example>
+      <ExampleCode>
+```css file=./examples/ColorsExample.css
+
+```
+      </ExampleCode>
+    </Example>
+    </View>
+  </TabItem>
+</Tabs>
+
+## Semantic
+
+### Font
+<TokenList namespace="colors" childNamespace="font" />
```

Comment:
Since _.mdx_ files doesn't have TS support, was thinking that it might be helpful to make some utility components to use in this file, example:

```
// in TokensList.tsx
// contrived name :)
const TokensColorsFont = () => <TokenList namespace="colors" childNamespace="font" />;

// usage here
<TokensColorsFont />
```

What do you think?

Reply:
Yeah, I like that. Makes it easier to try out the types (especially if we expand these). I updated that in the latest.